release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -1486,6 +1486,14 @@ As a result, the controller handles errors during migration better.
 [id="ocp-release-note-cluster-version-operator-bug-fixes_{context}"]
 === Cluster Version Operator
 
+[id="ocp-release-note-dev-console-bug-fixes_{context}"]
+=== Dev Console
+
+* Before this update, some entries on the *Quick Starts* page displayed duplicate link buttons. With this update, the duplicates are removed, and the link buttons are correctly displayed. (link:https://issues.redhat.com/browse/OCPBUGS-60373[OCPBUGS-60373])
+
+* Before this update, the onboarding modal that displayed when you first logged in was missing visuals and images, which made it unclear and less likely to prompt you to start the tour. With this release, the missing elements are added to the modal. As a result, the onboarding experience provides complete visuals consistent with the overall console design. (link:https://issues.redhat.com/browse/OCPBUGS-57392[OCPBUGS-57392])
```

Comment:
If it weren't for the need to rebase, I might have let this pass. This phrase bothers me as it is largely irrelevant (as the bug would have been fixed regardless) and more so, how do we know the impact of the missing visuals and images? 
```suggestion
* Before this update, the onboarding modal that displayed when you first logged in was missing visuals and images, which made the modal messaging unclear. With this release, the missing elements are added to the modal. As a result, the onboarding experience provides complete visuals consistent with the overall console design. (link:https://issues.redhat.com/browse/OCPBUGS-57392[OCPBUGS-57392])
```